core/controllers/community_dashboard.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Controllers for the community dashboard page."""
+
+from constants import constants
+from core.controllers import acl_decorators
+from core.controllers import base
+from core.domain import opportunity_services
+import feconf
+import utils
+
+
+class CommunityDashboardPage(base.BaseHandler):
+    """Page showing the community dashboard."""
+
+    @acl_decorators.open_access
+    def get(self):
+        if not feconf.COMMUNITY_DASHBOARD_ENABLED:
+            raise self.PageNotFoundException
+        self.render_template('dist/community-dashboard-page.mainpage.html')
+
```

Comment:
Since this page is open access you can serve this page through app.yaml

Reply:
@vojtechjelinek, This needs to serve on the basis of COMMUNITY_DASHBORD_ENABLED, will that be possible to do with app.yaml?

Reply:
Okay, si let's add comment here when we release community (I presume we would be enabling it once and the keeping it enabled) to move it to app.yaml

Reply:
Created an issue at #7402 and added a comment as asked. (Done!)